src/components/tooltip/hover-tooltip.jsx

```diff
@@ -19,16 +19,18 @@ var HoverTooltip = React.createClass({
   },
 
   render() {
+    let {hasTooltip, renderTooltip, children, ...otherProps} = this.props;
+
     return (
-      <div onMouseMove={this._handleMouseMove}>
-        {this.props.hasTooltip &&
+      <div {...otherProps} onMouseMove={this._handleMouseMove}>
+        {hasTooltip &&
```

Comment:
This is a recommended pattern http://facebook.github.io/react/docs/transferring-props.html